x/payments/spec/05_events.md

```diff
@@ -0,0 +1,38 @@
+# Events
+
+The Payment module emits the following events:
+
+## EndBlocker
```

Comment:
All events were placed under "EndBlocker" title, but some of the events are not generated from the EndBlocker, but are generated by message handlers. Please take a look at the bonds module events, which are organised into either a message or EndBlocker.

Reply:
Added events under handler , payment_contract_authorisation &  create_subscription are kept under @Endblocker as it is one time activity.